### PR TITLE
Drum machine

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: src/GlobalStyle.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    box-sizing: border-box;
+  }
+  body {
+    margin: 0;
+    background-color: #12121a;
+    color: #ccd;
+    font-family: "Poppins", -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
```
Comment (Quaquaro) [inline src/GlobalStyle.js:14]: Do your really need the webkit and the moz properties?

File: src/components/DrumPad.spec.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+
+describe('DrumMachine', () => {
+  it('renders a section with sixteen buttons', () => {
+    const padClick = jest.fn()
+    render(<DrumPad handleClick={padClick} />);
+
+    const drumPad = screen.getByRole('button', { name: 'Drum Pad' });
```
Comment (Quaquaro) [inline src/components/DrumPad.spec.js:11]: You only get the first button in the document. 
You could try this: expect(screen.getallByRole('button')).length.toBe(16).

File: src/components/DrumPad.spec.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    expect(drumPad).toBeInTheDocument();
+
+    userEvent.click(drumPad)
```
Comment (Quaquaro) [inline src/components/DrumPad.spec.js:14]: What do you want to test with it?
I would expect a second "it" here.

File: src/pages/DrumMachinePage.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      />
+      <DrumPad handleClick={handleClick} value={'./audio/Samples/Horn1.mp3'} />
+      <DrumPad
+        handleClick={handleClick}
+        value={'./audio/Samples/SynthShot1.wav'}
+      />
+      <DrumPad
+        handleClick={handleClick}
+        value="./audio/Samples/CongaFinger.wav"
+      />
```
Comment (Quaquaro) [inline src/pages/DrumMachinePage.js:21]: Maybe you can work with the map method to create the individual drum pads.
Maybe it is possible to store the values in an array and then map over them.

File: src/pages/DrumMachinePage.spec.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import DrumMachine from './DrumMachinePage';
+import { render } from '@testing-library/react';
+
+
+
+describe('DrumMachine', () => {
+  it('renders a section with sixteen buttons', () => {
+    render(<DrumMachine />);
+
```
Comment (Quaquaro) [inline src/pages/DrumMachinePage.spec.js:9]: I'm not sure, if you need a test here

**Review by Quaquaro — COMMENTED** [review]
Nice start, maybe you'll adjust the tests. I would be happy to help you.

File: src/GlobalStyle.js (hunk 1 above)
Comment (FaGru) [reply src/GlobalStyle.js:14]: No i don't really need that... thx 👍🏻

File: src/components/DrumPad.spec.js (hunk 2 above)
Comment (FaGru) [reply src/components/DrumPad.spec.js:11]: In this component i only have 1 button.
But you are right, i can test it on the DrumMachinePage.

Comment (FaGru) [reply src/components/DrumPad.spec.js:11]: In the DrumPad component I have only one button.
But you are right, I can test this on the DrumMachinePage

File: src/components/DrumPad.spec.js (hunk 3 above)
Comment (FaGru) [reply src/components/DrumPad.spec.js:14]: It tests, if a click on the button calls the function which plays the Sound.
Yes a second it is good here 😅

File: src/pages/DrumMachinePage.spec.js (hunk 5 above)
Comment (FaGru) [reply src/pages/DrumMachinePage.spec.js:9]: yes, it doesn't make much sense like that, but I'll test all the buttons here now, as already written above

File: src/pages/DrumMachinePage.js (hunk 4 above)
Comment (FaGru) [reply src/pages/DrumMachinePage.js:21]: I've already thought about that, but I think I'll do that later when I can change the samples on the buttons.

**Pull request merged**